src/scss/02_atoms/form/_autocomplete.scss

```diff
@@ -1,5 +1,5 @@
 $autocompleteConfig: (
-  paddingRight: rem(60px),
+  paddingRight: rem(40px),
```

Comment:
Diminuito il padding cosi da occupare sempre al massimo lo spazio dell'icona solamente senza andare a nascondere il testo in caso di size small